firmware/makefile

```diff
@@ -93,13 +108,24 @@ CFLAGS_WARNINGS_EXTRA = \
 	# -Wsign-conversion \
 	# -Wdouble-promotion \
 
-# These are a list of #define symbols in the makefile
+
+swageversion != if [ X"$(SET_SWADGE_VERSION)" = X"1" ]; then \
```

Comment:
What is this for? Echoing 0 and 1 don't seem very useful.

Reply:
Need to ask Richard

Reply:
OK, if `SET_SWADGE_VERSION` is 1, then `swageversion` will be 1, otherwise it will be 0. I'm going to clean this up.